release-notes.txt

```diff
@@ -1,3 +1,633 @@
+* native: Improve Documentation
+* CONTRIBUTING: cleanup and reorganize
+* core/msg_bus: enhance documentation
+* cpu/stm32: rewrite clock configuration documentation
+* doc: advertise forum.riot-os.org
+* doc: Give examples of what DEVELHELP does
+* gnrc/rpl: document implementation status
+* ieee802154/radio: indicate that FCS is ignored when reading buffer
+* uhcp: improve documentation
+* update board porting documentation page
+
+Build System / Tooling
+----------------------
++ testbed-support: add iotlab mapping for dwm1001
++ info.inc.mk: Add info-build-json as JSON-formatted build info
++ makefiles/utils: add util to compare to software versions
++ emulator: add debug feature with qemu emulator
+* Makefile.include: avoid recursive expansion of USEMODULE
++ makefiles/pseudomodules.inc.mk: netdev_ieee802154_legacy is a pseudomodule
+* dist/tools/bmp: Fix flashing with pygdbmi 0.10.0
+* drivers/at86rf215: fix dependency resolution for single radio
+* cpu/esp*: fix linker scripts
+* info-objsize: Restore info-objsize target
+* make: run static_tests.sh directly with `static-tests` target
+* picolibc: model as a feature
+* tools/esptool: add common makefile to define esptool variables
+* dist/tools: Use bash instead of sh where needed
+* tools/emulator: allow use with *term targets + introduce EMULATE=1 instead of emulate target
+* mips: adapt for toolchain update (2020.06-01)
+- remove the bootloader_% pseudo-feature, replace it with highlevel_stdio
+
+Continuous Integration
+----------------------
+
+* use Github actions for static tests instead of Travis
+    + Add support for Github annotations.
+* gh-actions/release-test: bump ssh-agent to v0.4.1
+
+Testing
+-------
++ tests/mpu_stack_guard: add automatic test script
++ tests/mtd_raw: add simple test for MTD
+* treewide: cleanup ENABLE_DEBUG
+* release-tests: fix artifact naming and steps cleanup for matrix build
+* tests/sys_atomic_utils_unittests: spice up test
+* tests/unittests: split out atomic_utils unittests
+* sys/shell: terminate shell on Ctrl-D
+
+Kconfig
+-------
++ Modules with support added:
+    + drivers/{a, b, c, d, e, f, g, h, i, j}*
+    + drivers/ili9341
+    + test/periph_*
++ cpu/stm32: add script to generate Kconfig.models and Kconfig.lines files
++ add modules to test on native
++ Clock configuration for:
+    + stm32l4
+    + stm32l5
+    + stm32wb
+    + stm32f0
+    + stm32g0g4
+    + stm32l0l1
+    + stm32f1f3
+* drivers/rn2xx3 : Kconfig updates.
+* MQTT-SN Clients: Kconfig Updates
+
+Examples
+--------
++ gnrc_networking: add option to enable DNS
+* gnrc_border_route: simplify ZEP setup on native
+* suit_update: use interactive sync in test
+* gcoap: Allow PUT/POST with empty payload
+* suit_update: update IETF draft reference in the README
+
+
+API Changes
+===========
+* periph_flashpage: Make pagewise API optional
+* sys/atomic_utils: Use volatile qualifier
+* net/eui_provider: API cleanup
+* sys/atomic_utils: change API to also return previous value
```

Comment:
Note: `sys/atomic_utils` is new in this release cycle. Shouldn't only API changes compared to the previous release be listed here. If so, those should be dropped.

```suggestion
* net/eui_provider: API cleanup
```

Reply:
sure! I missed that one.